recipe/emailpassword/recipe.go

```diff
@@ -71,7 +71,7 @@ func MakeRecipe(recipeId string, appInfo supertokens.NormalisedAppinfo, config *
 
 		mtRecipe := multitenancy.GetRecipeInstance()
```

Comment:
rejecting this PR cause the input to get user in all recipe will have tenantId anyway.